extensions/amp-story/1.0/animation.js

```diff
@@ -232,34 +238,42 @@ export class AnimationRunner {
   /**
    * Evaluates a preset's keyframes function using dimensions.
    * @param {!WebKeyframesDef|!WebKeyframesCreateFnDef} keyframesOrCreateFn
-   * @param {!Object<string, *>} keyframeOptions
+   * @param {!Object<string, *>=} keyframeOptions
    * @return {!Promise<!WebKeyframesDef>}
    * @private
    */
   resolvePresetKeyframes_(keyframesOrCreateFn, keyframeOptions) {
     if (typeof keyframesOrCreateFn === 'function') {
       return this.getDims().then((dimensions) => {
         const fn = /** @type {!WebKeyframesCreateFnDef} */ (keyframesOrCreateFn);
-        return fn(dimensions, keyframeOptions);
+        return fn(dimensions, keyframeOptions || {});
```

Comment:
Please use a default argument instead.
What makes it so it can now be undefined when it couldn't before?

Reply:
It was an optional argument at the Runner construction layer before.

I'd rather define it like this rather than a default value since it's only used optionally and we prevent creating an unused object for `<amp-story-animation>` this way.